src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -72,6 +73,7 @@ protected override void OnModelCreating(ModelBuilder builder)
 
         // Going forward use `IEntityTypeConfiguration` in the Configurations folder for managing
         // Entity Framework code first database configurations.
+        var eCache = builder.Entity<Cache>();
```

Comment:
🎨 Per the above comment let's use the separate configuration files. There's a few examples e.g. `Grant`.